test/functional/apps/management/ccs_compatibility/_data_views_ccs.ts

```diff
@@ -0,0 +1,96 @@
+
+  describe('data views ccs', function () {
+    before(async function () {
+      await security.testUser.setRoles(['kibana_admin']);
+    });
+
+    describe('index pattern wizard ccs', () => {
+      describe('remote cluster only', async () => {
+        this.beforeEach(async function () {
+          // delete .kibana index and then wait for Kibana to re-create it
+          await kibanaServer.uiSettings.replace({});
+          await PageObjects.settings.navigateTo();
+          await PageObjects.settings.clickKibanaIndexPatterns();
+        });
+        it('create index pattern using remote name', async () => {
+          await PageObjects.settings.createIndexPattern('ftr-remote:logstash*', null);
+          await PageObjects.settings.clickKibanaIndexPatterns();
+          const indexPatternList = await PageObjects.settings.getIndexPatternList();
+          expect(indexPatternList.length).to.eql(1);
+        });
+        it('create index pattern with wildcards in remote name', async () => {
+          await PageObjects.settings.createIndexPattern('*t*-remo*:log*', null);
+          await PageObjects.settings.clickKibanaIndexPatterns();
+          const indexPatternList = await PageObjects.settings.getIndexPatternList();
+          expect(indexPatternList.length).to.eql(1);
+        });
+
+        this.afterEach(async () => {
+          await PageObjects.header.waitUntilLoadingHasFinished();
+          await testSubjects.exists('indexPatternTable');
+          await kibanaServer.savedObjects.cleanStandardList();
+        });
+      });
+      describe('remote and local clusters', async () => {
+        before(async () => {
+          await es.transport.request({
+            path: '/blogs/_doc',
+            method: 'POST',
+            body: { user: 'cuffs', message: 20 },
+          });
+        });
+
+        this.beforeEach(async function () {
+          // delete .kibana index and then wait for Kibana to re-create it
+          await kibanaServer.uiSettings.replace({});
+          await PageObjects.settings.navigateTo();
+          await PageObjects.settings.clickKibanaIndexPatterns();
+        });
+        it('combined remote cluster and local cluster data view without wildcards', async () => {
+          await PageObjects.settings.createIndexPattern('blog*, ftr-remote:log*', null);
+          await PageObjects.settings.clickKibanaIndexPatterns();
+          const indexPatternList = await PageObjects.settings.getIndexPatternList();
+          expect(indexPatternList.length).to.eql(1);
+        });
+        it('combined remote cluster and local cluster data view with wildcards', async () => {
+          await PageObjects.settings.createIndexPattern('blog*, *t*-remo*:lo*', null);
+          await PageObjects.settings.clickKibanaIndexPatterns();
+          const indexPatternList = await PageObjects.settings.getIndexPatternList();
+          expect(indexPatternList.length).to.eql(1);
+        });
+
+        this.afterEach(async () => {
```

Comment:
And again lol.  Maybe your ide did this (autocomplete)?

Reply:
Fixed.